src/ripple/rpc/impl/RPCHelpers.h

```diff
@@ -114,10 +114,34 @@ parseRippleLibSeed(Json::Value const& params);
 std::pair<PublicKey, SecretKey>
 keypairForSignature(Json::Value const& params, Json::Value& error);
 
+/**
+ * API version numbers used in API version 1
+ */
 extern beast::SemanticVersion const firstVersion;
 extern beast::SemanticVersion const goodVersion;
 extern beast::SemanticVersion const lastVersion;
 
+/**
+ * API version numbers used in later API versions
+ *
+ * Requests with a version number in the range
+ * [APIVersionRangeLow, APIVersionRangeHigh] are supported.
+ *
+ * Network Requests without explicit version numbers use
+ * APIVersionIfUnspecified.
+ * Command line Requests use APIVersionCommandLine.
+ */
+
+constexpr unsigned APIInvalidVersion = 0;
+constexpr unsigned APIFirstVersion = 1;
+constexpr unsigned APIVersionSupportedRangeLow = 1;
+constexpr unsigned APIVersionSupportedRangeHigh = 1;
+constexpr unsigned APIVersionIfUnspecified = APIFirstVersion;
+constexpr unsigned APIVersionCommandLine = APIVersionSupportedRangeHigh;
+constexpr unsigned APIVersionRippledStartUp = APIVersionSupportedRangeHigh;
```

Comment:
Considering that these two values are always likely to use a fixed version of the API, I wonder if defining values for them makes sense. We should _probably_ just use `RPC::APIVersionSupportedRangeLow` directly in the places where this is used and remove `APIVersionCommandLine` and `APIVersionRippledStartUp`.

Reply:
I will remove APIVersionCommandLine and APIVersionRippledStartUp. 

Just to confirm, we will use APIVersionSupportedRangeLow as the command line request version. I can see the benefit of both "high" and "low.

Reply:
I can argue for either low or high. I'm fine with using whatever is appropriate and leave that up to your judgement.

Reply:
I removed `APIVersionCommandLine` and `APIVersionRippledStartUp`, and used "high" for command line requests' versions.